draft-ietf-mls-protocol.md

```diff
@@ -2392,7 +2392,11 @@ following criteria hold:
 * U is a descendant of P in the tree
 * The nodes between U and P in the tree are all blank
 * The `parent_hash` field of U is equal to the parent hash of P with copath
-  child S, where S is the child of P that is not on the path from U to P.
+  child S, where S is the child of P that is not on the path from U to P
+* The sibling of every node with a blank parent between U (included) and P
+  (excluded) has an empty resolution
```

Comment:
I would merge this with the bullet above about the intermediate nodes being blanks.  So something like:

> Each node between U and P in the tree is blank, and the "copath child" of P (its child that is not in the direct path of U) has an empty resolution